spec/td/command/query_spec.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+
+require 'spec_helper'
+require 'td/command/common'
+require 'td/command/list'
+require 'td/command/query'
+require 'td/client'
+require 'tempfile'
```

Comment:
With these requires, I can run only this spec like `bundle exec rake spec SPEC_OPTS="-fd" SPEC=spec/td/command/query_spec.rb`.